vue-app/src/views/VerifyLanding.vue

```diff
@@ -2,66 +2,80 @@
-          v-tooltip="{
-            content: `If you've previously donated to a CLR round, use the same wallet to bypass some BrightID steps`,
-            trigger: 'hover click',
-          }"
-          width="16px"
-          src="@/assets/info.svg"
-          class="info-icon"
-        />
-      </h2>
-      <ul>
-        <li>
-          BrightID – available on
-          <a
-            href="https://apps.apple.com/us/app/brightid/id1428946820"
-            target="_blank"
-          >
-            iOS</a
-          >
-          or
-          <a
-            href="https://play.google.com/store/apps/details?id=org.brightid"
-            target="_blank"
-            >Android</a
-          >
-        </li>
-        <li>An Ethereum wallet, with enough gas for two transactions</li>
-        <li>Access to Zoom or Google Meet</li>
-      </ul>
-      <links to="/about-sybil-resistance/">Why is this important?</links>
-      <div v-if="isRoundFullOrOver" class="warning-message">
-        The current round is no longer accepting new contributions. You can
-        still get BrightID verified to prepare for next time.
-      </div>
-      <div class="btn-container mt2">
-        <wallet-widget
-          v-if="!currentUser"
-          :isActionButton="true"
-          :fullWidthMobile="true"
-        />
-        <links v-if="currentUser" to="/verify/connect" class="btn-primary">
-          I have BrightID installed
-        </links>
-        <links to="/projects" class="btn-secondary">Go back</links>
+      <div class="content">
+        <div class="flex-title">
+          <h1>Prove you’re only using one account</h1>
+        </div>
+        <div class="subtitle">
+          We use BrightID to stop bots and cheaters, and make our funding more
+          democratic.
+        </div>
+        <h2>
+          What you'll need
+          <img
+            v-tooltip="{
+              content: `If you've previously donated to a CLR round, use the same wallet to bypass some BrightID steps`,
+              trigger: 'hover click',
+            }"
+            width="16px"
+            src="@/assets/info.svg"
+            class="info-icon"
+          />
+        </h2>
+        <ul>
+          <li>
+            BrightID – available on
+            <a
+              href="https://apps.apple.com/us/app/brightid/id1428946820"
+              target="_blank"
+            >
+              iOS</a
+            >
+            or
+            <a
+              href="https://play.google.com/store/apps/details?id=org.brightid"
+              target="_blank"
+              >Android</a
+            >
+          </li>
+          <li>An Ethereum wallet, with enough gas for two transactions</li>
+          <li>Access to Zoom or Google Meet</li>
+        </ul>
+        <links to="/about-sybil-resistance/">Why is this important?</links>
+        <div
+          v-if="
+            $store.getters.isRoundJoinPhase &&
+            !currentRound &&
+            !isRoundFullOrOver
```

Comment:
@pettinarip I dont think it will be sufficient to just have isRoundJoinPhase. isRoundJoinPhase is true even when there is a current round as I believe it represents join phase for joining as a recipient, not join phase as in pre current round, so this message would always be displaying then unless the round is full or over without the check that there isnt a currentRound.

Reply:
Oh right. That is because the phases could overlap sometimes. Got it now, thanks